pages/linux/light.md

```diff
@@ -0,0 +1,19 @@
+# light
+
+> CLI to control backlight.
```

Comment:
Perhaps ...`the backlight of your screen.`?

Reply:
adjusted